src/v3/src/util/theme.ts

```diff
@@ -100,9 +100,11 @@ export const mapMuiThemeFromBrand = (
   // TODO: OKTA-517723 temporary override until odyssey-react-mui theme borderRadius value is fixed
   odysseyTheme.shape.borderRadius = 4;
 
-  const odysseyThemeCopy = cloneDeep(odysseyTheme);
   odysseyTheme.direction = languageDirection;
 
+  // Do not modify `odysseyTheme` after this line, it will have no effect on the resulting theme
```

Comment:
There was no "bug" per-se but it would have resulted in a bug if we were using a version of Odyssey with components that read the `theme.direction` value.

Reply:
I was originally using this branch to test some Odyssey changes too but pivoted it to only handle the JS changes. Thought it would be good to let this in still since it needs to be fixed as it will result in real bugs in the future if forgotten